libs/src/services/creatorNode/index.js

```diff
@@ -417,6 +428,64 @@ class CreatorNode {
     this.authToken = null
   }
 
+  /**
+   * Gets the clock values across the replica set for the wallet in userStateManager.
+   *
+   * Returns clock values across replica set in the structure:
+   *
+   * {
+   *  type: 'primary' or 'secondary',
+   *  endpoint: <Content Node endpoint>,
+   *  wallet: wallet in userStateManager
+   *  clockValue: /users/clock_status/:wallet response OR null
+   * }
+   */
+  async getClockValuesFromReplicaSet () {
+    const user = this.userStateManager.getCurrentUser()
+    if (!user || !user.creator_node_endpoint) {
+      console.error('No user or Content Node endpoint found')
+    }
+
+    const primary = CreatorNode.getPrimary(user.creator_node_endpoint)
+    const secondaries = CreatorNode.getSecondaries(user.creator_node_endpoint)
+    const replicaSet = CreatorNode.getEndpoints(user.creator_node_endpoint)
+
+    const clockValueRequest = async (endpoint) => {
+      let type = ''
+      if (primary === endpoint) type = 'primary'
+      else if (secondaries.includes(endpoint)) type = 'secondary'
+
+      try {
+        const resp = await axios({
+          method: 'get',
+          baseURL: endpoint,
+          url: `/users/clock_status/${user.wallet}`
+        })
+
+        return {
+          type,
+          endpoint,
+          wallet: user.wallet,
+          clockValue: resp.data.data.clockValue
+        }
+      } catch (e) {
```

Comment:
the immediate usage of this method is for mad-dog/helpful for future utility. I think returning null is okay because I rather return null than throw an error. there could be some network interference, or maybe the nodes in the replica set were busy at the time